src/Directions.ts

```diff
@@ -1,18 +1,43 @@
+import { firstArrayValues, lastArrayValues } from './web/utils';
+
 const RIGHT = 1;
 const LEFT = 2;
 const UP = 4;
 const DOWN = 8;
 
+// public interface
 export const Directions = {
   RIGHT: RIGHT,
   LEFT: LEFT,
   UP: UP,
   DOWN: DOWN,
+} as const;
+
+// internal interface
+export const CompositeDirections = {
+  RIGHT: RIGHT,
+  LEFT: LEFT,
+  UP: UP,
+  DOWN: DOWN,
   UP_RIGHT: UP | RIGHT,
   DOWN_RIGHT: DOWN | RIGHT,
   UP_LEFT: UP | LEFT,
   DOWN_LEFT: DOWN | LEFT,
 } as const;
 
+export const axialDirectionsList = firstArrayValues(
+  Object.values(CompositeDirections),
+  4
+);
+
+export const diagnalDirectionsList = lastArrayValues(
```

Comment:
This is unused

Reply:
This is an error, `diagnalDirectionsList` should be used, but i accidently used `axialDirectionsList` in it's place.